builtins/amp-video.js

```diff
@@ -36,7 +36,31 @@ export function installVideo(win) {
     }
 
     /** @override */
+    prerenderAllowed() {
+      return true;
+    }
+
+    /** @override */
+    buildCallback() {
+      /** @private */
+      this.realPreloadAttrValue_ = this.element.getAttribute('preload');
+      const posterAttr = this.element.getAttribute('poster');
+
+      if (!posterAttr) {
+        console/*OK*/.warn(
```

Comment:
Do we really want to spam readers with warnings? Should it be protected by devmode?